components/n-ui/tracking/index.js

```diff
@@ -8,6 +10,6 @@ module.exports = {
 		});
 	},
 	scrollDepthComponents: require('./ft/events/scroll-depth-components'),
-	scrollDepth: require('./ft/events/scroll-depth')
-
+	scrollDepth: require('./ft/events/scroll-depth'),
+	view: oTracking.view
```

Comment:
Out of interest, why does this need to live in n-ui? Can we not use the method directly  from `o-tracking` in the various apps?

Asking as we're potentially bringing in additional weight here to apps that may not require this functionality.

Reply:
Although it does actually look like this is happening already... just my lack of knowledge of n-ui 🙈